### PR TITLE
Added some pagination responsibility to infinite_scroll_grid

### PR DESCRIPTION
**Pull request by sarahscott**
Fixes https://github.com/artsy/eigen/issues/1984

The `infinite_scroll_grid` was assuming the first page of artworks would be passed in as the `artworks` prop and would start its `page` state at 1. With this fix, the grid will start at page 0 if no artworks are passed in as props.

simulator screen shot nov 15 2016 5 33 47 pm

### CODE REVIEW

File: Example/Emission/ARRootViewController.m [hunk 1]
```diff
@@ -119,7 +119,7 @@ - (ARCellData *)jumpToHomepage
 - (ARCellData *)jumpToGene
 {
   return [self tappableCellDataWithTitle:@"Gene" selection: ^{
-    id viewController = [[ARGeneComponentViewController alloc] initWithGeneID:@"contemporary-pop"];
+    id viewController = [[ARGeneComponentViewController alloc] initWithGeneID:@"website"];
```
Comment (sarahscott) [inline Example/Emission/ARRootViewController.m]: Not all that relevant; can revert to contemporary pop if need be

File: Example/Podfile.lock [hunk 2]
```diff
@@ -87,9 +87,9 @@ EXTERNAL SOURCES:
   Artsy+UIFonts:
     :git: https://github.com/artsy/Artsy-UIFonts.git
   Emission:
-    :path: ../
+    :path: "../"
```
Comment (sarahscott) [inline Example/Podfile.lock]: Does this seem right?

File: Example/Emission/ARRootViewController.m (hunk 1 above)
Comment (orta) [inline Example/Emission/ARRootViewController.m]: e.g. these

File: Example/Podfile.lock (hunk 2 above)
Comment (orta) [inline Example/Podfile.lock]: as it's the only change, it's probably worth reverting

Comment (alloy) [inline Example/Podfile.lock]: Agreed.

**Comment by alloy** [comment]
Can you add a CHANGELOG entry?

**Pull request merged**